mlir/lib/Dialect/QC/Translation/qasm3/QASM3Emitter.cpp

```
@@ -0,0 +1,1477 @@
+    return success();
+  }
+
+  LogicalResult floatDecl(SMLoc /*loc*/, StringRef id,
+                          const Expr* initializer) {
+    if (initializer != nullptr) {
+      return assignFloat(id, *initializer);
+    }
+    return success();
+  }
+
+  LogicalResult boolAssign(SMLoc /*loc*/, StringRef id,
```

Cppcheck (2.21.0)

[style] The function 'boolAssign' is never used.

(unusedFunction)